com.unity.render-pipelines.core/Runtime/Lighting/ProbeVolume/ProbeReferenceVolume.cs

```diff
@@ -295,20 +295,21 @@ internal class Cell
             public SphericalHarmonicsL2[] sh;
             public float[] validity;
             public int minSubdiv;
-
-            [System.NonSerialized]
-            public int flatIdxInCellIndices = -1;
-
-            [System.NonSerialized]
-            public bool loaded = false;
+            public int indexChunkCount;
```

Comment:
Do we need these 2 serialized?

Reply:
I currently set them at bake time. I feel like it should stay there to be able to keep compatibility if we change values etc.